util/multirun.sh

```diff
@@ -23,9 +23,41 @@ set -o pipefail
 
 usage()
 {
-    echo >&2 "usage: $0 <path/to/cip/binary> [<path/to/manifest.yaml>,<path/to/service-account.json>, ...]"
-    echo >&2 "The 2nd argument onwards are '<manifest>,<service-account>,<service-account>...' strings."
-    echo >&2
+cat <<EOF >&2
+usage: $0 <path/to/cip/binary> [<MANIFEST>,<KEYFILE>[,<KEYFILE>,...] ...]
+
+MANIFEST: path/to/manifest.yaml
+KEYFILE: path/to/service-account.json
+
+The 2nd argument onwards are
+
+    '<manifest>,<service-account>,<service-account>...'
+
+strings.
+
+If an execution does not require any service account, pass in '<manifest>,'
+(notice the trailing comma).
+
+If you only want to run against manifests that changed in a Git commitish,
+specify the CIP_GIT_REV and CIP_GIT_DIR environment variables. E.g.,
+CIP_GIT_REV=HEAD
+CIP_GIT_DIR=/path/to/git/repo
+
+EOF
+}
+
+# Filter the given list of files by checking if it is a Manifest file. Files
+# must be absolute paths.
+collect_manifests()
+{
+    local cip="$1"
+    shift 1
+
+    for f in "$@"; do
+        if "$cip" -parse-only -manifest="$f"; then
```

Comment:
Is it really OK to just ignore args that don't parse?  Why is that not an error?

Reply:
It's OK to ignore because it just removes all files that are not parseable as promoter manifests. It's  "filter(...)" in Python.